File: Week 1/mario-more.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, row, col, space;

    // Get Height from user
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (row = 0; row < height; row++)
    {
        for (space = 0; space < height - row - 1; space++)
        {
            printf(" ");
        }
        for (col = 0; col <= row; col++)
        {
            printf("#");
        }
        printf("  ");
        for (col = 0; col <= row; col++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Week 6/mario-more.py
from cs50 import get_int

height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

# Loop through rows and columns

for i in range(height):
    for space in range(height - i - 1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")
    print("  ", end="")
    for k in range(i+1):
        print("#", end="")
    print()
